sonar-dotnet-shared-library/src/main/java/org/sonarsource/dotnet/shared/plugins/GeneratedFileFilter.java

```diff
@@ -26,8 +26,11 @@
 import org.sonarsource.dotnet.shared.plugins.protobuf.FileMetadataImporter;
 import org.sonarsource.dotnet.shared.plugins.protobuf.ProtobufImporters;
 
+import javax.swing.text.StyledEditorKit;

```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused import 'javax.swing.text.StyledEditorKit'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AUselessImportCheck)